### ISSUE TITLE
where is mscomm ?

### ISSUE THREAD

**Issue by jiangge12** [BAndysc/AvaloniaVisualBasic6#18]
with mscomm , AvaloniaVisualBasic6 will be real useful for some simple iot project

**Comment by leiming-star** [comment]
yes you right,i away use mscommand in visual basic to connect my siemens plc
